yetanotherco/aligned_layer
# fix: infinite tasks scripts

# Description

- This PR fixes the infinite tasks make targets.

# To Test
- Run `make batcher_send_infinite_sp1` and `batcher_send_infinite_groth16` and make sure they are working correctly
